roles/0-init/tasks/hostname.yml

```diff
@@ -17,16 +22,7 @@
   # 2021-08-31: Periods in /etc/hostname fail with some WiFi routers (#2904)
   # command: hostnamectl set-hostname "{{ iiab_hostname }}.{{ iiab_domain }}"
 
-#- name: Install /etc/sysconfig/network from template (redhat)
-#  template:
-#    src: roles/network/templates/network/sysconfig.network.j2
-#    dest: /etc/sysconfig/network
-#    owner: root
-#    group: root
-#    mode: 0644
-#  when: is_redhat
-
-# roles/network/tasks/hosts.yml [no longer in use] ALSO did this:
+# should the first entry match just hostname and domain move to after localhost?
```

Comment:
> should  [ `/etc/hosts` ] first entry match just hostname and domain move to after localhost?

> line: '127.0.0.1     {{ iiab_hostname }}.{{ iiab_domain }} localhost.localdomain localhost {{ iiab_hostname }} box box.lan'

Good question (I don't know what's best).  Post another PR if possible?

Reply:
Your ball game now, you figure it out.

Reply:
#1815 is the starting point of the thread.

Reply:
Recap:

1. #1815 ***was solved by...***
   - PR #1847

2. #2904 ***was solved by...***
   - PR #2973

Reply:
The '127.0.0.1 box.lan hack in /etc/hosts' was more to ease the need for rewriting documentation, where what would be needed to be entered in the browser's address bar 'localhost' on the iiab box with a desktop vs 'box' or 'box.lan' from a client on the LAN could be the same was the excuse that was made for the hack at that point of time in the past.